x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authz/store/CompositeRolesStore.java

```diff
@@ -630,29 +631,6 @@ void merge(MergeableIndicesPrivilege other) {
             }
         }
 
-        private static void collatePrivilegesByRemoteIndices(
-            final RemoteIndicesPrivileges[] remoteIndicesPrivileges,
-            final boolean allowsRestrictedIndices,
-            final Map<Set<String>, Map<Set<String>, MergeableIndicesPrivilege>> remoteIndicesPrivilegesMap
-        ) {
-            assert remoteIndicesPrivileges != null;
-            // if a remote index privilege is an explicit denial, then we treat it as non-existent to stay consistent with local index
-            // privileges
-            final boolean isExplicitDenial = remoteIndicesPrivileges.length == 1
-                && "none".equalsIgnoreCase(remoteIndicesPrivileges[0].indicesPrivileges().getPrivileges()[0]);
-            if (isExplicitDenial) {
-                return;
-            }
-            for (final RemoteIndicesPrivileges remoteIndicesPrivilege : remoteIndicesPrivileges) {
-                final Set<String> clusterAlias = newHashSet(remoteIndicesPrivilege.remoteClusters());
-                collatePrivilegesByIndices(
```

Comment:
The diff here is due to inlining `collatePrivilegesByIndices` -- I'm inlining because the method is now only used in one place and as such it's unnecessary to have it be stand-alone.